lib/game/Waiting.jsx

```diff
@@ -20,7 +20,7 @@ const Waiting = ({ playerId, host, game }) => {
             <Text style={styles.title}>Tipsy</Text>
             <Text style={styles.text}>Waiting for opponent</Text>
             <Text style={styles.text}>Invite link :</Text>
-            <Text style={styles.text}>
+            <Text style={styles.text} accessibilityLabel="Join link">
```

Comment:
I'd rather add an id on the Text above this one and an `accessibilityLabelledBy="idAbove"` on this one

Reply:
If I put id on this Text component (react-native-web),  I don't see it in the resulting html...

Reply:
see https://necolas.github.io/react-native-web/docs/text/, you have to use `nativeID`